### PR TITLE
Better messaging on route not found in get_op_for_request

### PR DESCRIPTION
**Pull request by atmz**
Fixes https://github.com/striglia/pyramid_swagger/issues/162 (or at least one of the cases there). Should not have any effect on performance. Also corrects docstring.

### CODE REVIEW

**Comment by djeebus** [comment]
This would have saved me a good amount of time over the last few weeks!

File: pyramid_swagger/tween.py [hunk 1]
```diff
@@ -561,6 +561,11 @@ def get_op_for_request(request, route_info, spec):
         op = spec.get_op_for_request(request.method, route_path)
```
Comment (striglia) [inline pyramid_swagger/tween.py]: I find this logic flow slightly counter-intuitive. Can we just move the lowest-level raise up into this if block?

File: tests/tween_test.py [hunk 2]
```diff
@@ -214,7 +214,7 @@ def test_get_op_for_request_not_found_when_route_has_no_path():
     swagger_spec = Mock(spec=Spec)
     with pytest.raises(PathNotMatchedError) as excinfo:
         get_op_for_request(request, route_info, swagger_spec)
-    assert 'Could not find a matching Swagger operation' in str(excinfo.value)
+    assert 'Could not find a matching route' in str(excinfo.value)
```
Comment (striglia) [inline tests/tween_test.py]: Maybe update the func name. Existing one is pretty mechanical (as opposed to explaining the likely programming mistake)

**Comment by striglia** [comment]
🎉

**Pull request merged**